python/pyspark/mllib/linalg.py

```diff
@@ -387,9 +434,60 @@ def __init__(self, size, *args):
                 if self.indices[i] >= self.indices[i + 1]:
                     raise TypeError("indices array must be sorted")
 
+    def copy(self):
```

Comment:
My comments about DenseVector methods carry over to copy + other SparseVector methods